src/core/id-headers.js

```diff
@@ -20,8 +28,15 @@ export function run(conf) {
       id = h.parentElement.id || h.id;
     }
     if (!conf.addSectionLinks) continue;
-    h.appendChild(html`
-      <a href="${`#${id}`}" class="self-link" aria-label="§"></a>
+    let label = h.closest(".appendix")
+      ? l10n.perma_for_appendix
+      : l10n.perma_for_section;
+    const sectionNumber = h.querySelector(":scope > bdi.secno");
+    if (sectionNumber) {
```

Comment:
What are the cases where the `bdi.secno` might not exist?

Reply:
Anything that is "introductory" (e.g., "Abstract", "Status of This Document").